js/wizard.js

```diff
@@ -0,0 +1,437 @@
+function validateIPaddress(ipaddress) {
```

Comment:
If the function returns just a **boolean** then why not just return directly what the Regex Test method returns (it either returns [true or false](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test))?

This way you avoid duplicated conditions

```javascript
if (validateIPaddress('192.168.0.1'))
```
and 
```javascript
if (/^(25[0-5]|2[0-4][0-9]|[...])
```